config/HEADER

```diff
@@ -1,13 +1,15 @@
-Copyright ${year} original authors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
+/*
+ * Copyright 2017-20\d\d original authors
```

Comment:
Used a year regex here to allow anything `20xx` wasn't sure what the game plan was to have to update all files every year or not but this relaxes that requirement. If we want to have it be explicitly 2019 and when we are ready to switch to 2020 just change it that works too. Just was going for a little flexibility there if that was not the case.

Reply:
I reverted this change because it was causing `Missing header in: xxx` errors and when running `licenseFormat` the updated licenses had `2017-20\d\d` in them, so something is not working right

Reply:
@graemerocher i just configured Checkstyle since it was part of the PR builds and that was the problem i was trying to fix without realizing what actually used that HEADER file was the licenseFormat plugin. 

I think its still going to be broken even after your revert. When I ran `licenseFormatMain` I ended up with licenses that were commented twice. 😬  And that would be b/c I now have comments in that file that were not there to begin with. 

From initial glance at it, probably best path forward would be to put the old HEADER file back that used `${year}` and let the licenseFormat plugin use that and create a new one just for checkstyle that uses the `20\d\d` format. I can get a PR open some time today with that change if that solution sounds reasonable.